test/JDBC/upgrade/13_4/schedule

```diff
@@ -85,6 +85,14 @@ babel_trigger
 nested_trigger_inside_proc
 nested_trigger_with_dml
 triggers_with_transaction
+Test-sp_addrole-dep
+Test-sp_addrolemember-dep
```

Comment:
dep test files shouldn't be added to schedules. After Narwhal release they should be added to 14_6 schedule, and all schedules after that.